creator/creator.go

```diff
@@ -82,6 +87,14 @@ func (c *Creator) SetOutlineTree(outlineTree *model.PdfOutlineTreeNode) {
 	c.externalOutline = outlineTree
 }
 
+// SetPageLabels adds the specified page labels to the PDF file generated
+// by the creator. See section 12.4.2 "Page Labels" (p. 382 PDF32000_2008).
+// NOTE: for existing PDF files, the page label ranges object can be obtained
+// using the model.PDFReader's GetPageLabels method.
+func (c *Creator) SetPageLabels(pageLabels core.PdfObject) {
```

Comment:
Sorry I missed the review period. It seems like the type should be core.PdfObjectDictionary since the PageLabels object is a dictionary.

Reply:
core.PdfObjectDictionary implements core.PdfObject so this works fine.  One can also put the dictionary inside an indirect object (core.PdfIndirectObject) which is actually common especially if the dictionary gets big, so it makes sense to use core.PdfObject here as it is more generic.